Tools/CMake/AMReX_Config.cmake

```diff
@@ -104,12 +104,6 @@ function (configure_amrex)
          AMREX_NVCC_MAJOR_VERSION=${NVCC_VERSION_MAJOR}
          AMREX_NVCC_MINOR_VERSION=${NVCC_VERSION_MINOR} )
 
-      set_target_properties( amrex
-         PROPERTIES
-         CUDA_SEPARABLE_COMPILATION ON      # This adds -dc
-         CUDA_RESOLVE_DEVICE_SYMBOLS OFF
```

Comment:
Summary:
- we intentionally remove `CUDA_RESOLVE_DEVICE_SYMBOLS OFF` for the `amrex` target since @mic84 tested this and it is superfluous.
- we intentionally do not set it downstream either in `setup_target_for_cuda_compilation` because this did not build for WarpX